taxi/models.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    class Mete:
+        ordering = ("name", )
+
+    def __str__(self) -> str:
+        return str(self.name)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="cars"
+    )
+
+    drivers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name="cars"
+    )
+
+    class Mete:

```

Comment:
There is a typo in the class name `Mete`. It should be `Meta` to correctly define the model's metadata.